awx/main/migrations/0109_workflowjobnode_template_node.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 2.2.8 on 2020-03-06 14:53
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('main', '0108_v370_unifiedjob_dependencies_processed'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='workflowjobnode',
```

Comment:
Is this going to be valuable/meaningful if we don't backfill it for existing data?

Reply:
We can't backfill the existing data. That's the point, if you don't save it at this point, recovery of it is hopeless.

Reply:
Yea, I guess introducing this _now_ is better than nothing?  If we built CLIs or tooling that relied on this data, it would mean it wouldn't really work with older data.